s4e-backend/src/main/java/pl/cyfronet/s4e/listener/GroupListener.java

```diff
@@ -20,6 +22,9 @@
     private final TemplateEngine templateEngine;
     private final MailService mailService;
 
+    @Value("${front.web.base.url}")
```

Comment:
Shouldn't this be `mail.urlDomain`? It's the same, and it's used in the same context of mail-sending.